draft-ietf-ohai-ohttp.md

```diff
@@ -1057,7 +1057,7 @@ might need an arrangement with Oblivious Relay Resources. This arrangement might
 be necessary to prevent having the large volume of requests being classified as
 an attack by the server.
 
-If a server accepts a larger volume of requests from a relay, it needs to
+If a gateway accepts a larger volume of requests from a relay, it needs to
```

Comment:
I think you should change all instances here, rather than just the first.  Probably the next paragraph too.  


But ... we say above "servers that operate the Oblivious Gateway Resource", which I think is crucial here.  This uses "server" because it is about protecting the server infrastructure, not a specific resource.  So I think we should revert this one.